k9mail-library/src/main/java/com/fsck/k9/mail/store/webdav/WebDavStore.java

```diff
@@ -47,6 +61,7 @@
  * and email information.
  * </pre>
  */
+@SuppressWarnings("deprecation")
```

Comment:
I'm still not a fan of this. If we're using deprecated stuff we want to see the warning. Getting rid of the deprecated stuff is the way to get to 0 warnings, not disabling warnings :)